exonum-java-binding-core/src/main/java/com/exonum/binding/service/BlockCommitedEventImpl.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.service;
+
+import com.exonum.binding.storage.database.Snapshot;
+import java.util.Optional;
+
+public class BlockCommitedEventImpl implements BlockCommittedEvent {
```

Comment:
I'd replace with `@AutoValue` implementation (this particular implementation is a value) — so that it gets `#toString`.